coretex/cli/commands/node.py

```diff
@@ -179,6 +180,18 @@ def config(advanced: bool) -> None:
     activateAutoUpdate()
 
 
+@click.command()
+@click.option("--tail", "-n", type = int, help = "Shows N last logs.")
+@click.option("--follow", "-f", is_flag = True, help = "Displays logs realtime.")
+@click.option("--timestamps", "-t", is_flag = True, help = "Displays timestamps for logs.")
+def logs(tail: Optional[int], follow: bool, timestamps: bool) -> None:
+    if getNodeStatus() in [NodeStatus.inactive, NodeStatus.deleted]:
```

Comment:
Well we have a requirement for node commands that docker must be started, did this requirement change in the meantime? Should I exclude logs command checking for Docker existance? If docker is not started there is no possible way to check if container exists even tho it could exist just docker can be turned off?

Reply:
I asked for checking if a docker _container_ of the Node is running, not whole docker service

Reply:
Apologies for misunderstanding, thanks for clarification.